apps/test/unit/templates/HourOfCodeGuideEmailDialogTest.js

```diff
@@ -0,0 +1,32 @@
+import {assert, expect} from 'chai';
+import React from 'react';
+import {UnconnectedHourOfCodeGuideEmailDialog as HourOfCodeGuideEmailDialog} from '@cdo/apps/templates/HourOfCodeGuideEmailDialog';
+import {shallow} from 'enzyme';
+import i18n from '@cdo/locale';
+import experiments from '@cdo/apps/util/experiments';
+
+describe('HourOfCodeGuideEmailDialog', () => {
+  const defaultProps = {
+    signedIn: true,
+  };
+
+  it('renders signed out text in Dialog', () => {
+    experiments.setEnabled(experiments.HOC_TUTORIAL_DIALOG, true);
+    const wrapper = shallow(<HourOfCodeGuideEmailDialog {...defaultProps} />);
+    expect(wrapper.contains(i18n.emailMeAGuide()));
+  });
+
+  it('renders correct translations if user is not signed in', () => {
+    experiments.setEnabled(experiments.HOC_TUTORIAL_DIALOG, true);
+    const wrapper = shallow(
+      <HourOfCodeGuideEmailDialog {...defaultProps} signedIn={false} />
+    );
+    expect(wrapper.contains(i18n.getGuideContinue()));
+  });
+
+  it('renders null if experiment flag is false', () => {
```

Comment:
We will eventually modify the above tests and remove this one when we go out from behind the experiment flag.